test/android/core/javatests/src/org/xwalk/core/xwview/test/DownloadListenerTest.java

```diff
@@ -10,8 +10,10 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import org.apache.http.HttpRequest;
 import org.chromium.base.test.util.Feature;
 import org.chromium.net.test.util.TestWebServer;
+import org.xwalk.core.internal.XWalkCookieManagerInternal;
```

Comment:
We should use ```org.xwalk.core.XWalkCookieManager``` since test case was in xwalk core instead of xwalk core internal.